client/my-sites/woocommerce/woop/style.scss

```diff
@@ -22,78 +22,92 @@ body.is-section-woocommerce-installation.theme-default.color-scheme {
-	@include woocommerce-colors();
-	margin-top: 100px;
-	padding-top: 60px;
-	padding-bottom: 40px;
-	padding-left: 20px;
-    padding-right: 20px;
-
-	// on desktop make the section full width so it takes the whole space
-	@media ( min-width: 660px ) {
-		margin-left: -30px;
-		width: calc( 100% + 20px );
-	}
 
-	background-color: var( --color-woocommerce-onboarding-background );
-	.formatted-header .formatted-header__title {
+	.woop__landing-page-features-section {
+		@include woocommerce-colors();
+		background-color: var( --color-woocommerce-onboarding-background );
+		margin-top: 100px;
+		padding-top: 60px;
+		padding-bottom: 40px;
 
-		@media ( max-width: $break-mobile ) {
-			margin-left: 0;
-			margin-right: 0;
-			width: 100%;
-			max-width: fit-content;
-			text-align: center;
+		.formatted-header .formatted-header__title {
+			@include onboarding-font-recoleta;
+			text-align: left;
+			font-size: 2rem;
+			max-width: 377px;
+			line-height: 40px;
+			padding-left: 0; // Match promo-card pad-left
 		}
 
-		text-align: left;
-		margin-left: 32px;
-		font-size: 2rem;
-		max-width: 377px;
-		line-height: 40px;
-		@include onboarding-font-recoleta;
-	}
+		.promo-section {
+			.promo-section__promos .promo-card {
+				box-shadow: none;
+				background-color: var( --color-woocommerce-onboarding-background );
 
-	.promo-section__promos .promo-card {
-		// 3 column layout
-		@media ( min-width: $break-wide ) {
-			width: calc( 33% - 1em );
-		}
-		box-shadow: none;
-		background-color: var( --color-woocommerce-onboarding-background );
+				.action-panel__title {
+					justify-content: normal;
+					font-weight: 500;
+					font-size: $font-body;
 
-		.action-panel__title {
-			@media ( max-width: $break-mobile ) {
-				display: block;
-				width: 100%;
-				text-align: center;
+					.gridicon {
+						padding-right: 0.5rem;
+						fill: #0675c4; // TODO: maybe we need to define this color as a variable somewhere?
+					}
+				}
+
+				.action-panel__body {
+					font-size: $font-body-small;
+				}
 			}
-			font-weight: 500;
-			font-size: 1.25rem;
 		}
 
-		.action-panel__body {
-			@media ( max-width: $break-mobile ) {
-				text-align: center;
-			}
+		// Extend grey background outside the bottom stripe
+		&::before {
+			content: '';
+			background-color: var( --color-woocommerce-onboarding-background );
+			position: absolute;
+			height: 100%;
+			width: 200vw;
+			left: -100vw;
+			z-index: -1;
+			margin-top: -60px;
 		}
+	}
+
+	// 3 column layout + 1 column layout
+	.woop__landing-page-features-section {
+		padding-left: 10%;
+		padding-right: 5%;
 
-		.gridicon {
-			fill: #0675c4; // TODO: maybe we need to define this color as a variable somewhere?
+		.promo-section {
+			.promo-section__promos .promo-card {
+				padding-left: 0;
+				padding-right: 0;
+				width: 90%;
+
+				@media ( min-width: $break-wide ) {
+					width: calc( 30% );
+					padding-left: 0;
+					padding-right: 30px;
+				}
+			}
```

Comment:
@Rolilink  / @allilevine I pulled both the background and column layout rules out from the other presentation rules to make things easier to reason about. I don't think there was anything wrong with the previous implementation I just couldn't figure out how it worked and needed to fix margins/alignment at various breakpoints in the previous commit [`3b583a6` (#60838)](https://github.com/Automattic/wp-calypso/pull/60838/commits/3b583a6ba537bf015f3811370fe0193c1df3d70a). I tried modifying the padding/margins we had but couldn't quite figure out how they worked so ended up with these percentage based margins/padding instead.